sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -0,0 +1,116 @@
+                  
+## 3. User-visible Level-Zero backend selection (and the default backend)
+
+The Level-Zero backend is added as a first-class citizen to the cl::sycl::backend enumeration:
+
+``` C++
+enum class backend : char { host, opencl, level_zero, all };
+```
+
+There are multiple ways in which the Level-Zero backend can be selected by the user.
+        
+### 3.1 Through an environment variable
+        
+The SYCL_DEVICE_FILTER environment variable limits the SYCL runtime to use only a subset of the system's devices.
+By using ```level_zero``` for backend in SYCL_DEVICE_FILTER you can select the use of Level-Zero as a SYCL backend.
+For further details see here: [https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md](https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md).
+        
+### 3.2 Through a programming API
+        
+There is an extension that introduces a filtering device selection to SYCL described in
+[https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/FilterSelector/FilterSelector.adoc](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/FilterSelector/FilterSelector.adoc).
+Similar to how SYCL_DEVICE_FILTER applies filtering to the entire process this device selector can be used to
+programmatically select the Level-Zero backend.
+                
+When neither the environment variable nor the filtering device selector are used, the implementation chooses
+the Level-Zero backend for GPU devices supported by the installed Level-Zero runtime.
+The serving backend for a SYCL platform can be queried with the ```get_backend()``` member function of ```cl::sycl::platform```.
+
+## 4. Interoperability with the Level-Zero API
+
+This chapter describes the various interoperability possible between SYCL and Level-Zero.
+The application must include both of the following headers in order to use any of the interoperation APIs described in this section,
+and they must be included in the order shown:
+
+``` C++
+  #include "level_zero/ze_api.h"
+  #include "sycl/backend/level_zero.hpp"
+```
+### 4.1 Mapping of SYCL objects to Level-Zero handles
+
+These SYCL objects encapsulate the corresponding Level-Zero handles:
+|  SYCL object  |    Level-Zero handle   |
+|-------------|:------------|
+|platform |ze_driver_handle_t|
+|device   |ze_device_handle_t|
+|context  |ze_context_handle_t|
+|queue    |ze_command_queue_handle_t|
+|program  |ze_module_handle_t|
+
+### 4.2 Obtaining of native Level-Zero handles from SYCL objects
+                
+The ```get_native<cl::sycl::backend::level_zero>()``` member function is how a raw native Level-Zero handle can be obtained
+for a specific SYCL object. It is currently supported for SYCL platform, device, context, queu, event and program classes.

```

Comment:
```suggestion
for a specific SYCL object. It is currently supported for SYCL platform, device, context, queue, event and program classes.
```

Also, maybe use backquotes here to format the class type names as code?

Reply:
OK